reactome_analysis_datasets/reactome_analysis_datasets/dataset_fetchers/grein_fetcher.py

```diff
@@ -168,10 +170,29 @@ def _get_sample_ids(self, list_metadata) -> list:
             sample_id_list.append(key)
         return sample_id_list
 
-    def get_available_datasets(self, no_datasets: int) -> list:
+    def get_available_datasets(self, no_datasets: int = None) -> list:
```

Comment:
Is this function still needed? I guess this is now part of the `overview_fetcher.py` in the API searcher package?